### PR TITLE
Automated versioning

### PR DESCRIPTION
**Pull request by andoulla**
Documenting the approach that we will take in versioning and which tools are going to be used

### CODE REVIEW

**Comment by andoulla** [comment]
@thomasdigby @hollg

**Comment by hollg** [comment]
This looks great to me. I think strictly enforcing rules for versioning, commit messages etc will save us a lot of hassle in the long run and it sounds like this setup will do that!

**Review by thomasdigby — APPROVED** [review]
Awesome, I think this will be really great, looking forward to trying it out when we have our first alpha release ready 👍

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,7 @@
 node_modules
 dist
 .cache
+.eslintcache
+yarn-error.log
+storybook-static
+storybook-static
```
Comment (andoulla) [inline .gitignore:7]:
```suggestion

```

**Pull request merged**